components/Table/Table.tsx

```diff
@@ -14,7 +14,7 @@ import Pagination from './Pagination';
 export type TablePalette = 'primary';
 
 const CARD_HEIGHT = '40px';
-const MAX_PAGE_ROWS = 10;
+const MAX_PAGE_ROWS = 12;
```

Comment:
There is a UI bug where react table pagination doesn't play nice with how we remove columns from the table if there's an `activeTab`. Would like to resolve this issue eventually, but going to increase max page rows for now.

Reply:
hmm... ok, let's add this to the list of issues to look into

Reply:
#521

Reply:
why even use pagination when we have scrolling?